app/controllers/concerns/rendering.rb

```diff
@@ -49,6 +49,16 @@ def serializer
       raise NotImplementedError
     end
 
-    def includes; end
+    def includes_from_params
+      params[:include] ? params[:include].split(',') : []
+    end
+
+    def includes
+      return nil unless defined?(self.class::INCLUDES)
+
+      [nil, includes_from_params].flatten.map do |field|
+        self.class::INCLUDES[field.try(:to_sym)]
+      end
```

Comment:
:+1: It makes sense

Reply:
What would happen if `params[:include]` would get repeated includes, like `include=rules,rules,rules,...`?
What would happen if the param would get invalid or empty entry?

Reply:
Sorry :see_no_evil: somehow I got 2 commits instead of one, should be okay now.